packages/flame_forge2d/example/lib/balls.dart

```diff
@@ -60,39 +60,63 @@ class Ball extends BodyComponent {
       }
     }
   }
+
+  @override
+  void beginContact(Contact contact) {
+    final otherObject = contact.bodyA.userData == this
+        ? contact.bodyB.userData
+        : contact.bodyA.userData;
+
+    if (otherObject is Wall) {
```

Comment:
It should be possible to define what types that you want to collide with so that you don't get callbacks that you aren't interested in.

Reply:
I will look into this. I think it might be possible using the `preSolve` method.